src/mscorlib/shared/System/Guid.Unix.cs

```diff
@@ -0,0 +1,43 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+
+namespace System
+{
+    partial struct Guid
+    {
+        // This will create a new guid.  Since we've now decided that constructors should 0-init,
```

Comment:
> Since we've now decided that constructors should 0-init

Is this comment from, like, 1999? :smile:

Reply:
No idea :-) Came from CoreRT as well